modules/layers/src/scatterplot-layer/scatterplot-layer.ts

```diff
@@ -185,6 +186,11 @@ export default class ScatterplotLayer<DataT = any, ExtraPropsT extends {} = {}>
     model?: Model;
   };
 
+  // TODO why NonNullable? Why is `typeof picking.uniforms` `PickingUniforms` in luma
+  // but `PickingUniforms | undefined` here?
+  // uniformStore = new UniformStore<{picking: typeof picking.uniforms}>({picking});
+  uniformStore = new UniformStore<{picking: NonNullable<typeof picking.uniforms>}>({picking});
```

Comment:
@ibgreen any idea here? I can't get the same pattern working [as in luma](https://github.com/visgl/luma.gl/blob/master/examples/showcase/instancing/app.ts#L186)